bin.js

```diff
@@ -626,12 +626,15 @@ async function getAskMessage (dirname) {
     # Asking for help with: ${basename}
     # Please enter your question below. There is no specific formatting, but
     # feel free to use Markdown if you need to.
+    #
+    # You may have been dropped into vim. If you're unfamiliar with it, do:
+    # - press 'i' to enter insert mode
```

Comment:
Can we open Vim in insert mode, e.g. `+star`? I imagine frequent Vim users would see that they're in insert mode, and everyone else could just start typing, no surprises.

If so, we could replace all this handholding with one conditional line, "When finished, press Escape then `:wq` to save and exit" if it's Vim that is being opened (I think you said we can detect this?)